### PR TITLE
Input

### PR DESCRIPTION
**Pull request by ttahm3d**
- Added normal input boxes
- Input with validations
- Added radio buttons , Checkboxes and dropdown
- Added codeblocks for all these

### CODE REVIEW

**Review by AnkitKarnAK — COMMENTED** [review]
Everything looks perfect. Great work🔥🚀

File: docs/input/index.html [hunk 1]
```diff
@@ -76,7 +82,246 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
...
+    <input type="checkbox" name="brand" id="Asus" />
+    <label for="Asus">Asus</label>
+  </div>
+</div>
+            </code></pre>
+          </div>
+
+          <h4 class="heading-4">Dropdown</h4>
+          <div class="component-description">
+            Dropdown&apos;s are used to dislpay List of Values. The user can
```
Comment (shuntzhere) [inline docs/input/index.html:294]: Display is spelled incorrectly.

File: docs/input/index.html [hunk 2]
```diff
@@ -76,7 +82,246 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
...
+          <div class="component-description">
+            <p class="justify">
+              Input is another way the user interacts with websites. They are
+              used to get user&apos; inputs and choices
+            </p>
+          </div>
+          <h4 class="heading-4">Input Fields</h4>
+          <div class="component-description">
+            <p class="justify">
+              Input fileds usually have a label along with the actual input
```
Comment (shuntzhere) [inline docs/input/index.html:96]: Spelling mistake here.

File: docs/input/index.html [hunk 3]
```diff
@@ -76,7 +82,246 @@ <h2 class="heading-3 head-lvl-3" id="available-components">
...
+    <label for="Asus">Asus</label>
+  </div>
+</div>
+            </code></pre>
+          </div>
+
+          <h4 class="heading-4">Dropdown</h4>
+          <div class="component-description">
+            Dropdown&apos;s are used to dislpay List of Values. The user can
+            select on of the values for a particular property
```
Comment (shuntzhere) [inline docs/input/index.html:295]: I believe here would be one* of the

**Review by shuntzhere — COMMENTED** [review]
Some spelling mistakes and looks like some divs are not indented properly.

**Review by Sameony — APPROVED** [review]
Could'nt find any ambiguity or irrelevance, very nice code 👍🏻

File: docs/input/index.html (hunk 3 above)
Comment (ttahm3d) [reply docs/input/index.html]: Thanks for pointing

File: docs/input/index.html (hunk 2 above)
Comment (ttahm3d) [reply docs/input/index.html]: Thanks for pointing

File: docs/input/index.html (hunk 1 above)
Comment (ttahm3d) [reply docs/input/index.html]: Thanks for pointing

**Review by shuntzhere — APPROVED** [review]

**Pull request merged**